pygments/lexers/arturo.py

```diff
@@ -0,0 +1,312 @@
+import re
```

Comment:
standard file header is missing...

Reply:
- [x] [add file header](https://github.com/pygments/pygments/pull/2259/commits/a0ff99bc43bd98f1dd96f43afffe6cadf9d08829)